pom.xml

```diff
@@ -294,9 +295,8 @@
                             <Bundle-SymbolicName>org.mariadb.jdbc</Bundle-SymbolicName>
                             <Automatic-Module-Name>org.mariadb.jdbc</Automatic-Module-Name>
                             <Export-Package>org.mariadb.jdbc</Export-Package>
-                            <Import-Package>
-                                javax.naming,javax.management,javax.net;resolution:=optional,javax.net.ssl;resolution:=optional,javax.sql,javax.transaction.xa;resolution:=optional,org.slf4j;resolution:=optional
-                            </Import-Package>
+                            <Import-Package>org.osgi.framework,javax.naming,javax.management,javax.net;resolution:=optional,javax.net.ssl;resolution:=optional,javax.sql,org.osgi.service.jdbc;resolution:=optional,javax.transaction.xa;resolution:=optional,org.slf4j;resolution:=optional</Import-Package>
```

Comment:
I think you missed javax.security in this list. It prevents the driver from making SSL connections from an OSGi environment.

Reply:
I just added what was required for the jdbc service, if you think more packages are required I think you can simply open a new PR as this one is already closed.